<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickPeriodWidget</class>
 <widget class="QWidget" name="PickPeriodWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnPickPeriod">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick Custom Period</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxMonth">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxYear">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectionHint">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
